edb/pgsql/metaschema.py

```diff
@@ -4134,6 +4134,130 @@ def __init__(self) -> None:
         )
 
 
+class FTSParseQueryFunction(dbops.Function):
```

Comment:
Is this going to be fast enough? I guess there's no way around having to do this kind of computation, if we're doing a string input query.

Reply:
I'd bet that looping instead of doing recursion would help, though?